Content/Console/README.md

```diff
@@ -40,14 +43,42 @@ $ ./build.sh  // on unix
 - `DISABLE_COVERAGE` Will disable running code coverage metrics.  AltCover can have [severe performance degradation](https://github.com/SteveGilham/altcover/issues/57) so it worth disabling when looking to do a quicker feedback loop.
   - `DISABLE_COVERAGE=1 ./build.sh`
 
-### Watch Tests
 
-The `WatchTests` target will use [dotnet-watch](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/dotnet-watch.md) to watch for changes in your lib or tests and re-run your tests on all `TargetFrameworks`
+---
+
+### Building
+
 
 ```sh
-./build.sh WatchTests
+> build.cmd <optional buildtarget> // on windows
+$ ./build.sh  <optional buildtarget>// on unix
 ```
 
+---
+
+### Build Targets
+
+
+- `Clean` - Cleans up artifact and temp directories
+- `DotnetRestore` - Runs [dotnet restore](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x) on the sln file
```

Comment:
would `the sln file` here and below be more clear as `the solution`?

Reply:
Yeah.  Do you think I should link to official docs on what a `sln` file is as well?

Reply:
it's kinda on the "people probably already know" level, but it can't hurt :)

Reply:
done